model-client/src/commonMain/kotlin/org/modelix/model/IncrementalBranch.kt

```diff
@@ -326,6 +330,14 @@ class IncrementalBranch(val branch: IBranch) : IBranch, IBranchWrapper {
             modified(UnclassifiedNodeDependency(this@IncrementalBranch, childId)) // see .containsNode
         }
 
+        override fun setConcept(nodeId: Long, concept: IConceptReference?) {
+            val oldParentId = transaction.getParent(nodeId)
+            val oldRole = transaction.getRole(nodeId)
+            transaction.setConcept(nodeId, concept)
+            modified(ChildrenDependency(this@IncrementalBranch, oldParentId, oldRole))
```

Comment:
Could we get a test for this incremental support?
It would help to understand why `ChildrenDependency` is needed.